spec/models/policy_version_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe PolicyVersion do
+  describe '.current' do
+    let!(:previous) do
+      # create a policy before to make sure it doesn't get returned
+      PolicyVersion.create \
+        policy: policy('previous'),
+        role: owner,
+        policy_text: '[]'
+
+      # force constraints run
+      PolicyVersion.db.execute """
+        SET CONSTRAINTS ALL IMMEDIATE;
+        SET CONSTRAINTS ALL DEFERRED;
+      """
+    end
+
+    it 'returns nil if no policy load is in progress' do
+      expect(PolicyVersion.current).to_not be
+    end
+
+    it 'returns the currently loading policy version' do
+      pv = PolicyVersion.create \
+        policy: policy('current'),
+        role: owner,
+        policy_text: '[]'
+      expect(PolicyVersion.current).to eq pv
+    end
+  end
+
+  it "commits previously active policy version" do
```

Comment:
What does `commits` mean here, since it's not really committing the transaction?

Reply:
Finalization of the policy version load. I'll push a commit shortly which renames it so it'll hopefully become clearer.